workflow/tests/base_results/results_bills.csv

```diff
@@ -352,20 +353,27 @@ base-misc-bills-pv-detailed-only.xml,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,687.33,
 base-misc-loads-none.xml,1316.76,144.0,821.86,965.86,144.0,206.9,350.9,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 base-misc-neighbor-shading.xml,1659.63,144.0,1185.01,1329.01,144.0,186.62,330.62,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 base-misc-shielding-of-home.xml,1648.76,144.0,1200.54,1344.54,144.0,160.22,304.22,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 base-misc-usage-multiplier.xml,2604.02,144.0,1689.52,1833.52,144.0,492.06,636.06,0,0,0,0,60.24,60.24,74.2,74.2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-base-pv-battery-ah.xml,762.7,144.0,1195.91,455.0,144.0,163.7,307.7,0,0,0,0,0,0,0,0,-884.9,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-base-pv-battery-garage.xml,732.14,144.0,1151.88,412.52,144.0,175.62,319.62,0,0,0,0,0,0,0,0,-883.36,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-base-pv-battery-lifetime-model.xml,764.47,144.0,1195.91,456.77,144.0,163.7,307.7,0,0,0,0,0,0,0,0,-883.14,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-base-pv-battery.xml,762.7,144.0,1195.91,455.0,144.0,163.7,307.7,0,0,0,0,0,0,0,0,-884.9,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+base-pv-battery-ah.xml,752.38,144.0,1195.91,444.68,144.0,163.7,307.7,0,0,0,0,0,0,0,0,-895.23,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+base-pv-battery-garage.xml,720.28,144.0,1151.87,400.64,144.0,175.64,319.64,0,0,0,0,0,0,0,0,-895.22,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+base-pv-battery-lifetime-model.xml,760.84,144.0,1195.91,453.14,144.0,163.7,307.7,0,0,0,0,0,0,0,0,-886.77,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+base-pv-battery-round-trip-efficiency.xml,752.38,144.0,1195.91,444.68,144.0,163.7,307.7,0,0,0,0,0,0,0,0,-895.23,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
```

Comment:
This has the same simulation results as `base-pv-battery.xml`. Is it working?

Reply:
I'll try exaggerating the round trip efficiency, to say 0.8.

Reply:
I already tried an extreme value of 0.15, the results didn't change.

Reply:
Am I remembering correctly that you were requesting an additional output at one point when this value was not 1.0?

Reply:
@shorowit: Thanks for testing exaggerated values! I was expecting with a minor change you might not see much change in results since the losses were already small, but with big changes it ought to have an impact. There is one other place we can change the round trip efficiency that I can test out right now.

Reply:
Requesting an additional output when round trip efficiency was not 1.0?